frontend/packages/videosdk-rtc-component/src/newUi/components/AppMain.tsx

```diff
@@ -32,6 +33,15 @@ export function AppMain({ meetingTitle, defaultSideView }: AppMainProps) {
     medicalNoteSections.find((e) => e.hasNotication)
   );
 
+  const callTime = useRef(Date.now() + 1000 * 60 * 30);
+  const formatter: CountdownProps['formatter'] = (value) => {
+    // @ts-ignore
+    const minutes = Math.floor(value / 1000 / 60);
+    // @ts-ignore
+    const seconds = Math.floor((value / 1000) % 60);
+    return `${minutes.toString().padStart(2, '0')}:${seconds.toString().padStart(2, '0')}`;
+  };
```

Comment:
This being here is going to make everything re-render every second. Let's move this to a child component to return the Countdown component.

Also, let's do proper typing or type-casting to avoid the ts-ignore if possible.